src/wallet/api/pending_transaction.cpp

```diff
@@ -262,6 +262,17 @@ std::vector<std::string> PendingTransactionImpl::signersKeys() const {
     return keys;
 }
 
+std::vector<std::string> PendingTransactionImpl::dest_addresses() const {
+    std::vector<tools::wallet2::pending_tx> txs = m_pending_tx;
+    std::vector<std::string> result;
+    for(tools::wallet2::pending_tx tx : txs) {
+       for (cryptonote::tx_destination_entry dest : tx.dests) {
```

Comment:
Why copies ? THe code below does not seem to need them.

Reply:
It's helpful to be able to grab this information easily so I could do [this](https://github.com/monero-project/monero-gui/pull/3319/files#diff-7e31b62f1a469ab821efff1a907d18aca3ce0689488f0439d706304dc02e2a65R184) in monero-project/monero-gui#3319